datalad/distributed/create_sibling_gitlab.py

```diff
@@ -395,6 +407,9 @@ def _proc_dataset(refds, ds, site, project, remotename, layout, existing,
             "Unknown site access '{}' given or configured, "
             "known ones are: {}".format(access, known_access_labels))
 
+    pathsep = ds.config.get("datalad.gitlab-default-pathseparator", None) or "-"
+    project_stub = \
+        ds.config.get("datalad.gitlab-default-projectname", None) or "project"
```

Comment:
what are people's feelings about dataset-level configs versus e.g., global configs? I personally tend to forget configuring this for all levels in a dataset. But we could also just add an example "for-each-dataset" call that would take care of this in the doc(string).

Reply:
I assume this is a general comment that has nothing to do with this line, right?

If so: I thinking making it work again is good enough for this PR, and for now.